migrate_to_single_container.rb

```diff
@@ -68,11 +68,19 @@ def copy_all_documents
     logger.debug "Full listing: #{listing}"
 
     if listing
+
+      # skip user when there are more files than we can list
+      if listing.split("\n").size > 9999
+        File.open('log/10k_users.log', 'a') { |f| f.puts username }
+        raise "User has too many files"
```

Comment:
Exceptions get caught, logged and the migration flag gets set back from "in_progress" to "not_started", so that the user's storage uses the shared container normally again.
Also the username is put in a list of failed migrations.